inputremapper/configs/preset.py

```diff
@@ -232,21 +232,12 @@ def get_mapping(
             existing = self._mappings.get(permutation)
             if existing is not None:
                 return existing
-
-        logger.error(
-            "Combination %s not found. Available: %s",
-            repr(combination),
-            list(
-                self._mappings.keys(),
-            ),
-        )
-
         return None
 
     def dangerously_mapped_btn_left(self) -> bool:
         """Return True if this mapping disables BTN_Left."""
-        if InputCombination([InputConfig.btn_left()]) not in [
-            m.input_combination for m in self
+        if InputConfig.btn_left().type_and_code not in [
```

Comment:
I still get the warning `WARNING input_config.py:268: No origin_hash set for {'type': 1, 'code': 272}` here

```suggestion
        if (ecodes.EV_KEY, ecodes.BTN_LEFT) not in [
```